### ISSUE TITLE
Issue on page /indexing/cliDocker/README.html

### ISSUE THREAD

**Issue by arnounesco** [iodepo/odis-arch#237]
the init of gleaner will pull https://raw.githubusercontent.com/earthcubearchitecture-project418/gleaner/master/configs/demo.yaml but this gives back a 404...

**Comment by fils** [comment]
Good catch, this whole page really is completely replaced by the work in https://github.com/gleanerio/archetype  and https://github.com/gleanerio/archetype/blob/master/docs/quickstart.md 

The GeoBon group wants this in particular and have been chatting with me this last week about it.   So I will get this updated.

Thanks
Doug
